exposed-tests/src/test/kotlin/org/jetbrains/exposed/sql/tests/shared/entities/OrderedReferenceTest.kt

```diff
@@ -0,0 +1,139 @@
+import org.jetbrains.exposed.sql.tests.shared.assertTrue
+import org.junit.Test
+
+class OrderedReferenceTest : DatabaseTestsBase() {
+    object Users : IntIdTable()
+
+    object UserRatings : IntIdTable() {
+        val value = integer("value")
+        val user = reference("user", Users)
+    }
+
+    object UserNullableRatings : IntIdTable() {
+        val value = integer("value")
+        val user = reference("user", Users).nullable()
+    }
+
+    class UserRatingDefaultOrder(id: EntityID<Int>) : IntEntity(id) {
+        companion object : IntEntityClass<UserRatingDefaultOrder>(UserRatings)
+
+        var value by UserRatings.value
+        var user by UserDefaultOrder referencedOn UserRatings.user
+    }
+
+    class UserNullableRatingDefaultOrder(id: EntityID<Int>) : IntEntity(id) {
+        companion object : IntEntityClass<UserNullableRatingDefaultOrder>(UserNullableRatings)
+
+        var value by UserNullableRatings.value
+        var user by UserDefaultOrder optionalReferencedOn UserNullableRatings.user
+    }
+
+    class UserDefaultOrder(id: EntityID<Int>) : IntEntity(id) {
+        companion object : IntEntityClass<UserDefaultOrder>(Users)
+
+        val ratings by UserRatingDefaultOrder referrersOn UserRatings.user orderBy UserRatings.value
+        val nullableRatings by UserNullableRatingDefaultOrder optionalReferrersOn UserNullableRatings.user orderBy UserNullableRatings.value
+    }
+
+    @Test
+    fun testDefaultOrder() {
+        withOrderedReferenceTestTables {
+            val user = UserDefaultOrder.all().first()
+
+            unsortedRatingValues.sorted().zip(user.ratings).forEach { (value, rating) ->
+                assertEquals(value, rating.value)
+            }
+            unsortedRatingValues.sorted().zip(user.nullableRatings).forEach { (value, rating) ->
+                assertEquals(value, rating.value)
+            }
+        }
+    }
+
+    class UserRatingMultiColumn(id: EntityID<Int>) : IntEntity(id) {
+        companion object : IntEntityClass<UserRatingMultiColumn>(UserRatings)
+
+        var value by UserRatings.value
+        var user by UserMultiColumn referencedOn UserRatings.user
+    }
+
+    class UserNullableRatingMultiColumn(id: EntityID<Int>) : IntEntity(id) {
+        companion object : IntEntityClass<UserNullableRatingMultiColumn>(UserNullableRatings)
+
+        var value by UserNullableRatings.value
+        var user by UserMultiColumn optionalReferencedOn UserNullableRatings.user
+    }
+
+    class UserMultiColumn(id: EntityID<Int>) : IntEntity(id) {
+        companion object : IntEntityClass<UserMultiColumn>(Users)
+
+        val ratings by UserRatingMultiColumn referrersOn UserRatings.user orderBy listOf(UserRatings.value to DESC, UserRatings.id to DESC)
+        val nullableRatings by UserNullableRatingMultiColumn optionalReferrersOn UserNullableRatings.user orderBy listOf(
+            UserNullableRatings.value to DESC,
+            UserNullableRatings.id to DESC
+        )
```

Comment:
Could you confirm if it is technically also possible to chain this operator, like:
```kt
val nullableRatings by UserNullableRatingMultiColumn optionalReferrersOn UserNullableRatings.user
        orderBy UserNullableRatings.value to SortOrder.DESC
        orderBy UserNullableRatings.id to SortOrder.DESC
```

And will the result be cumulative (so identical to using `listOf()`), like the existing `orderBy()` behavior?